Source/Libraries/FaultData/DataOperations/LSCVSDataOperation.cs

```diff
@@ -86,6 +86,7 @@ public override void Execute(MeterDataSet meterDataSet)
                 TableOperations<DbDisturbance> disturbanceTable = new TableOperations<DbDisturbance>(connection);
 
                 IEnumerable<Event> fileGroupEvents = eventTable.QueryRecordsWhere("FileGroupID={0}", meterDataSet.FileGroup.ID);
+                IEnumerable<EventType> allowedEventTypes = new TableOperations<EventType>(connection).QueryRecordsWhere("Name={0} AND Category={1}", "Sag", "PQ");
```

Comment:
`Category` is a UI only thing and can be changed by the user. WE probabyl should just use `Name`

Reply:
Did not realize this, fixed